src/biblenotify/core/notifications.py

```diff
@@ -92,11 +94,29 @@ def loadChapter(self, location: str) -> dict:
             return ["", ""]
 
         contents = QTextStream(file)
-        contents_string = contents.readAll()
+        contentsString = contents.readAll()
 
-        contents_json = json.loads(contents_string)
+        verse = self.currentVerse["verse"]
+        contentsJson = json.loads(contentsString)
+
+        chapter = contentsJson
+        chapterPlace = chapter["read"][0]["chapter"]
+
+        highlightedChapter = self.highlightVerse(verse, chapter)
 
         return {
-            "text": contents_json["read"][0]["text"],
-            "place": contents_json["read"][0]["chapter"]
+            "text": highlightedChapter,
+            "place": chapterPlace
         }
+
+    def highlightVerse(self, verse, chapter):
+        chapterText = chapter["read"][0]["text"]
+        
+        verseMatch = re.search(re.escape(verse), chapterText)
+        match = verseMatch.span()
+       
+        toHighlight = chapterText[match[0] : match[1]]
+
+        highlightChapter = chapterText.replace(toHighlight, "<b>" + toHighlight + "</b>")
+
+        return highlightChapter
```

Comment:
See above comment regarding the variable name

Also please add a newline to the end of the file